admin/sql/messaging/CreateTables.sql

```diff
@@ -0,0 +1,42 @@
+\set ON_ERROR_STOP 1
+
+BEGIN;
+
+SET search_path = 'messaging';
+
+CREATE TABLE edit_note_thanks (
+  thanker             INTEGER NOT NULL, -- PK, references editor.id
+  edit_note		        INTEGER NOT NULL, -- PK, references edit_note.id
```

Comment:
Might want to check your editor settings: several lines here have tab characters mixed in with spaces, leading to the weird indentation. :)